Changelog.md

```diff
@@ -15,6 +15,7 @@ Compiler Features:
 Bugfixes:
  * AST: Remove ``null`` member values also when the compiler is used in standard-json-mode.
  * Optimizer: Keep side-effects of ``x`` in ``byte(a, shr(b, x))`` even if the constants ``a`` and ``b`` would make the expression zero unconditionally. This optimizer rule is very hard if not impossible to trigger in a way that it can result in invalid code, though.
+ * Scanner: Fix bug where whitespace would be allowed within the ``->`` token (e.g. ``-   >`` becomes invalid now).
```

Comment:
Maybe need to be more specific with the example: `e.g. function f() -  > x {}` becomes invalid in Yul.